docs/requirements.md

```diff
@@ -99,7 +99,7 @@ The storage requirements recommended on the following table **depend mainly on t
 Codacy requires a database server to persist data that must satisfy the following requirements:
 
 -   The infrastructure hosting the database server must be provisioned with the hardware requirements described below
--   The DBMS server must be [PostgreSQL](https://www.postgresql.org/) **version 11.20**
+-   The DBMS server must be [PostgreSQL](https://www.postgresql.org/) **version 11.20**<!- TODO DOCS-612 Confirm version -->
```

Comment:
Since we now support 12, are we increasing the recommended version here? Any migration steps in case of breaking changes?

Reply:
Added version 12.* and set it as recommended.